### ISSUE TITLE
[Feature request] Allow to copy values from 'Detail view'

### ISSUE THREAD

**Issue by seniordevel** [lxqt/qps#486]
Often I need path or command-line values elsewhere and it is really a hassle to read/type them step by step.

Suggestion as simplest solution, for less effort and costs for the developer:
Add  'copy value to clipboard' action to each field's onclick event.

**Comment by tsujan** [comment]
Yes, it should have been done before, but we've been very busy with LXQt+Wayland lately.
